client/executor/src/native_executor.rs

```diff
@@ -269,6 +269,52 @@ impl sp_core::traits::ReadRuntimeVersion for WasmExecutor {
 	}
 }
 
+impl CodeExecutor for WasmExecutor {
+	type Error = Error;
+
+	fn call<
+		R: Decode + Encode + PartialEq,
+		NC: FnOnce() -> result::Result<R, Box<dyn std::error::Error + Send + Sync>> + UnwindSafe,
+	>(
+		&self,
+		ext: &mut dyn Externalities,
+		runtime_code: &RuntimeCode,
+		method: &str,
+		data: &[u8],
+		use_native: bool,
+		_native_call: Option<NC>,
+	) -> (Result<NativeOrEncoded<R>>, bool) {
+		if use_native {
+			log::warn!("`use_native` is set to `true` when calling the `WasmExecutor`. Ignoring.");
+		}
```

Comment:
And what should the user do with this?

Please remove this. It is not correct to print this here.